space-configs/demo/sismo.ts

```diff
@@ -28,6 +28,53 @@ export default {
     ],
   },
   apps: [
+    {
+      type: "zkDrop",
+      metadata: {
+        name: "Sismo Early Community Minting",
+        slug: "zk-drop-early-community",
+        description: "Prove you are an early member of Sismo Community to mint your Sismo Early Community NFT.",
+        tags: ["NFT"],
+        image: "sismo_app_store_zkdrop_early_community_500x500.png",
+        createdAt: new Date("2023-07-11T10:00"),
+        isTransferable: true,
+      },
+      sismoConnectRequest: {
+        appId: "0x5b7249cf5d8a1669cec21e5aa554299d",
+        authRequests: [{ authType: AuthType.VAULT }],
+        claimRequests: [{ groupId: "0xc90878eaa974c31bc62c52ad86121765", value: 1 },],
+        impersonateAddresses: ["0x8ab1760889F26cBbf33A75FD2cF1696BFccDc9e6"],
+      },
+      templateConfig: {
+        nftMetadata: {
+          name: "Sismo Early Community NFT",
+          description: "Claimable by the early community of Sismo, this NFT is a small gift to everyone that helped us to build Sismo before July 11 2023.",
+          image: "nft_sismo_early_community_1000x1000.png",
+          symbol: "SEarlyNFT",
+        },
+        chains: [
```

Comment:
I would add goerli or sepolia here as the relayer is not enabled it will be hard to test if you don't have the corresponding token

Reply:
@yum0e what do you prefer? goerli or sepolia?